defaults/main.yml

```diff
@@ -30,11 +28,10 @@ adminremove_passwords: False
 # List of users to remove passwords for
 admin_list_of_user_lists_remove_passwords: "{{ admin_list_of_user_lists }}"
 
-# If you need to point to a hard coded config file or otherwise mess with the
-# manual ssh which runs at the start of this role to test whether to use the
-# default user or root.
-# Defaults to ssh_args from ansible.cfg or empty string if that isn't set/found
-ssh_extra_args: "{{ ssh_args | default('') }}"
```

Comment:
The ssh_args does not come from ansible.cfg, the comment is wrong.

Reply:
One could argue that this is a breaking change. IMO users should not use ssh_args as a variable name, since it is too easy to confuse with the variable in ansible.cfg

Reply:
Who knows, maybe the comment might have been right for ancient Ansible 2.2, but in any case this looks like an improvement in Ansible 2.9 era.